content/docs/guide/map_submission/map_porting.md

```diff
@@ -9,322 +9,241 @@ weight: 2
+Some mappers provide their own screenshots on [gamebanana](https://gamebanana.com/) (Surf, Bhop, KZ) or [jump.tf](https://jump.tf/forum/) (RJ/SJ).  
+While you may not use those screenshots directly, feel free to recreate them using the steps above.
 
-- **Entity Editor** - Create and modify entities in the map
-- **Pakfile Explorer** - View and edit the pakfile as a file tree. It can be used to remove or pack assets, as well as automatically refactor references to assets in other lumps.
-- **Texture Browser** - View all textures in a map, similar to Hammer's texture browser. This is used for content review.
-- **Jobs** - Automated tasks that can be run on the BSP, such as:
-  - Remove official Valve assets
-  - Apply [Stripper](https://www.bailopan.net/stripper/) configs
-  - Batch replace textures
-- **Entity Review** - Lists invalid or problematic entities in the map
-- **Compressed BSP Saving** - Saves the BSP in a compressed format, which is required for map submissions
-- **Map Summary** - Reviews content in the BSP according to map submission requirements
+{{</hint>}}
+    
+## Step 2: Remove Valve Assets
+Maps may include packed materials, models, and other asset ( similar to a **.zip** file).  
+Momentum Mod [cannot legally redistribute](/guide/map_submission/map_submission/#source-assets) Valve's assets.  
+Valve's assets may still be used on maps however it is necessary to remove them from the **.bsp**.   
 
-![Lumper Example](/images/map_porting/lumper_example.png)
+1. Go to the **Jobs** tab in Lumper
+2. Add "Remove Game Assets" job
+3. Run the job
 
-To use Lumper, download the latest release from the [Lumper releases page](https://github.com/momentum-mod/lumper/releases). Lumper is a standalone application, so you can run it without installing anything. Just extract the ZIP file and run it from anywhere.
+{{<hint info>}}
 
-{{< hint info >}} For Windows users, you can run `RegisterLumperURLProtocol.ps1` with PowerShell to register the `lumper://` URL protocol. This allows you to open BSP files directly from dashboard map pages, which is useful for reviewing. {{< /hint >}}
+Momentum Mod automatically mounts assets from CS:S, CS:GO, and Portal 2 so players with those games installed will still be able to see them even after removal.  
+A small amount of assets from Source games not listed above can remain packed but **only** if they are **relatively generic** and **not fundamental** to that game's aesthetic.  
+Use the **Texture Browser** in Lumper to see which game each texture comes from.  
+For other assets check the **Required Games** tab.
 
-You can use Lumper to open any BSP file, modify it accordingly, and then save it.
+{{</hint>}}
+{{<hint warning>}}
+When keeping  Valve assets packed, it is required that you list them in the **porting changelog** during [Map Submission](/guide/map_submission/map_submission/)
+{{</hint>}}
```

Comment:
This seems directly related to the content of the previous box, but is shown as a separate box which makes it seem unrelated to the first one. It should just be one box.